packages/compass-collection/src/components/collection-header-actions/collection-header-actions.tsx

```diff
@@ -1,22 +1,29 @@
-import { Button, ButtonSize, css } from '@mongodb-js/compass-components';
+import {
+  Button,
+  ButtonSize,
+  css,
+  spacing,
+} from '@mongodb-js/compass-components';
 import React from 'react';
 
 import ViewInformation from './view-information';
 
+const editViewButtonStyles = css({
+  marginLeft: spacing[2],
```

Comment:
Maybe
```suggestion
    flexShrink: 0,
    marginLeft: spacing[2],
```

<img width="211" alt="Screen Shot 2022-04-20 at 6 07 05 PM" src="https://user-images.githubusercontent.com/1791149/164333219-d43a0008-e149-42b5-8550-64741484def0.png">
<img width="366" alt="Screen Shot 2022-04-20 at 6 08 32 PM" src="https://user-images.githubusercontent.com/1791149/164333220-17beb622-f266-4fad-b638-150383e88681.png">

Reply:
Sometimes I feel like doing a single PR that just puts `flex: none` on literally all the buttons in the app, I don't think we ever want them to collapse like that 😅